<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font-size: 15px;
	color: #FFFFFF;
	qproperty-alignment: AlignCenter;
	text-align: center; 
}

QMainWindow {
background-color: #333333;
}

QLineEdit {
	color: #FFFFFF;
	background-color: #444444;
	qproperty-alignment: AlignCenter;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="firstname">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFirstName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lastname">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLastName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_welcome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ok">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>211</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Open New Window</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
